src/azul/service/elasticsearch_service.py

```diff
@@ -430,7 +430,7 @@ def transform_summary(self,
         ):
             es_search.aggs.metric(
                 agg_name, 'cardinality',
-                field='{}.keyword'.format(cardinality),
+                field=f'{cardinality}.keyword',
```

Comment:
I think this should use concatenation instead. Is it okay if I make this change in the fixup commit?

Reply:
Yes.